.gitignore

```diff
@@ -11,4 +11,6 @@ pull_dev_database.sh
 
 snapshots
 .env*
-!.env.*.example
\ No newline at end of file
+!.env.*.example
+
+public/assets/
```

Comment:
ignoring precompiled assets. The dockerfile runs a command to pre-compile these for the deployed app